x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/jwt/JwtRealmSettings.java

```diff
@@ -229,6 +227,20 @@ public static ClientAuthenticationType parse(String value, String settingKey) {
         "client_authentication.shared_secret"
     );
 
+    // Individual Cache settings
+
+    public static final Setting.AffixSetting<TimeValue> JWT_CACHE_TTL = Setting.affixKeySetting(
+        RealmSettings.realmSettingPrefix(TYPE),
+        "jwt.cache.ttl",
+        key -> Setting.timeSetting(key, DEFAULT_JWT_CACHE_TTL, Setting.Property.NodeScope)
+    );
+
+    public static final Setting.AffixSetting<Integer> JWT_CACHE_MAX_USERS = Setting.affixKeySetting(
+        RealmSettings.realmSettingPrefix(TYPE),
+        "jwt.cache.max_users",
```

Comment:
This setting controls the number of entries in the cache, which may not be the number of users.
I think it should be
`jwt.cache.size` (like the DLS bitset and automata caches)
or `jwt.cache.max_keys` (like the api-key cache)

Reply:
Renamed to `jwt.cache.size`.